tests/bsim/bluetooth/host/att/pipeline/dut/Kconfig

```diff
@@ -0,0 +1,6 @@
+config BT_HCI_TX_PRIO
```

Comment:
can you add a comment saying why you need this file? It's probably to be able to set the prio, right?